docs/4-At-The-Internship/asking_questions.md

```diff
@@ -4,15 +4,17 @@ Ah, one of the most scary parts of the internship but also probably the most imp
 
 Ask lots of questions! Ask questions! Did I mention yet that you should ask people questions?
 
-Especially at first, since you probably don't know how most things work yet
+Especially at first, since you probably don't know how most things work yet.
 
 ### Who do you ask?
 
-usually you'd ask your supervisor
-but sometimes to the #engineering slack channel or whatever
-whoever would know the answer
-if you have 1:1s with your supervisor (and if you dont' I'd suggest to try to set them up) you can also ask every week if your questions are too much or not enough and if they're good questions and get advice on that
+Usually you'd ask your supervisor but sometimes to the #engineering slack channel or whatever whoever would know the answer if you have 1:1s with your supervisor (and if you dont' I'd suggest to try to set them up) you can also ask every week if your questions are too much or not enough and if they're good questions and get advice on that.
```

Comment:
Often you'd ask your supervisor, but it's also common for people to ask questions to the #engineering slack channel (or wherever the online-chat place is with people who'd know the answer). If you have 1:1s with your supervisor (and if you don't I'd suggest to try to set them up) you can also ask questions then. I also like to ask every week if I'm asking too many questions or not enough and if they're good questions and get advice on that.